volume/local/local.go

```diff
@@ -133,6 +140,7 @@ func (r *Root) scopedPath(realPath string) bool {
 	return false
 }
 
+// Volume holds a path associated with a unique name
```

Comment:
Volume implements the `Volume` interface from the `volume` package and represents the volumes created by `Root`